packages/react-button/src/components/Button/useButton.ts

```diff
@@ -14,14 +15,18 @@ export const useButton = (
   options: ComposePreparedOptions,
 ): ButtonState => {
   const buttonRef = useRef<HTMLButtonElement | null>(null);
-
   useImperativeHandle(props.componentRef, () => ({
     focus: () => buttonRef.current?.focus(),
   }));
   useFocusRects(ref as React.RefObject<HTMLElement>);
 
+  const buttonBehaviorProps = useButtonBehavior(props, buttonRef);
+  const { onKeyDown } = buttonBehaviorProps;
+
   return {
+    ...buttonBehaviorProps,
     ...props,
+    onKeyDown,
```

Comment:
This looks fragile as will require the same action across all components...

`useAccessibility()` has the same issue but handles it in a single place:
https://github.com/microsoft/fluentui/blob/346cd61df2ce2268f4126eab1b31a173904ff4c2/packages/fluentui/react-bindings/src/hooks/useAccessibility.ts#L68-L73

Reply:
You're right! I'm actually already doing this in place and just didn't tackle it correctly. Fixing it.